syncstorage/tests/functional/test_storage.py

```diff
@@ -72,17 +72,32 @@ def _switch_user(self):
         finally:
             self.root = orig_root
 
-    def test_get_collections(self):
+    def test_get_info_collections(self):
         # col1 gets 3 items, col2 gets 5 items.
         bsos = [{"id": str(i), "payload": "xxx"} for i in xrange(3)]
-        self.app.post_json(self.root + "/storage/col1", bsos)
+        resp = self.app.post_json(self.root + "/storage/col1", bsos)
+        ts1 = resp.json["modified"]
         bsos = [{"id": str(i), "payload": "xxx"} for i in xrange(5)]
-        self.app.post_json(self.root + "/storage/col2", bsos)
+        resp = self.app.post_json(self.root + "/storage/col2", bsos)
+        ts2 = resp.json["modified"]
         # only those collections should appear in the query.
         resp = self.app.get(self.root + '/info/collections')
         res = resp.json
         keys = sorted(res.keys())
         self.assertEquals(keys, ["col1", "col2"])
+        self.assertEquals(res["col1"], ts1)
+        self.assertEquals(res["col2"], ts2)
+        # Updating items in col2, check timestamps.
+        bsos = [{"id": str(i), "payload": "yyy"} for i in xrange(2)]
+        resp = self.app.post_json(self.root + "/storage/col2", bsos)
+        self.assertTrue(ts2 < resp.json["modified"])
+        ts2 = resp.json["modified"]
+        resp = self.app.get(self.root + '/info/collections')
+        res = resp.json
+        keys = sorted(res.keys())
+        self.assertEquals(keys, ["col1", "col2"])
+        self.assertEquals(res["col1"], ts1)
+        self.assertEquals(res["col2"], ts2)
```

Comment:
This was some extra sanity-checking I added to ensure /info/collections behaves as intended.